flow/activities/flowable_core.go

```diff
@@ -349,6 +349,16 @@ func syncCore[TPull connectors.CDCPullConnectorCore, TSync connectors.CDCSyncCon
 	activity.RecordHeartbeat(ctx, pushedRecordsWithCount)
 	a.Alerter.LogFlowInfo(ctx, flowName, pushedRecordsWithCount)
 
+	if a.OtelManager != nil {
+		currentBatchID, err := a.OtelManager.GetOrInitInt64Gauge(
```

Comment:
ideally we'd have something like I have for typed temoral slots & then this could just be
```
otel_metrics.CurrentBatchId.Record(ctx, a.OtelManager, res.CurrentSyncBatchID)
```